local_db/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.utils import timezone
+from ulid import ulid
+
+
+def local_request_id():
+    return str(ulid())
```

Comment:
Are these the IDs that will have to be extracted from L4 via eyeballs?  If so are ULIDs practical?

Reply:
So, I will once again (:D) argue that we do not need copy-pasteble IDs at all:

At any given them there is only one active[1] request for a workspace/user pair. So it is trivial for an output checker to view the correct release request, was they know both the user and workspace from the existing word doc.

[1] where active means state=SUBMITTED, which is coming Real Soon Now :tm: